src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelBinderProviderContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    /// <summary>
+    /// A context object for <see cref="IModelBinderProvider.Create(ModelBinderProviderContext)"/>.
```

Comment:
FYI there's no need to specify the parameter types unless you're dealing w/ a method group. Leaving off the parenthetical part reduces churn if we add parameters later.